scripts/generate_json_docs.py

```diff
@@ -16,15 +16,18 @@
 import inspect
 import json
 import os
-import sys
 import types
 
+import gcloud
 import pdoc
 from parinx.parser import parse_docstring
 from parinx.errors import MethodParsingException
 
 from verify_included_modules import get_public_modules
 
+ABSOLUTE_LIBRARY_PATH = '/'.join(os.path.abspath(gcloud.__file__)
+                                   .split('/')[:-2])
+
 
```

Comment:
I'm 👎  on the code as written and 👍  on @tseaver suggestion. Also, your above should've used `os.path.sep` instead of `/` and the period (`.`) line continuation (`.split`) should be avoided if not necessary (certainly not necessary here)